Assets/Scripts/PlayerMovement.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerMovement : MonoBehaviour
+{
+
+    public float moveSpeed = 5f;
+    public float gravity = 10f;
+    public float jumpSpeed = 5f;
+
+    private CharacterController charController;
+    
+
+    Vector3 velocity;
+    
+
+
+    // Start is called before the first frame update
+    void Start()
+    {
+        charController = GetComponent<CharacterController>();
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+
+        //Movement
+        float x = Input.GetAxis("Horizontal");
+        float z = Input.GetAxis("Vertical");
+
+        Vector3 move = transform.right * x + transform.forward * z;
+
+        charController.Move(move * moveSpeed * Time.deltaTime);
+
+        velocity.y -= gravity * Time.deltaTime;
+
+        charController.Move(velocity * Time.deltaTime);
+
+
+        //Jump
+        
+        if (charController.isGrounded)
+        {
+            if (Input.GetKeyDown(KeyCode.Space))
```

Comment:
I feel like you should flip the is grounded and the input's If statements but it works for now.

The way it is right now, you can only check for player input while grounded. That can be a problem if we add a feature where we want to check for spacebar input while not grounded. But like I said it works for now.